plugins/leemons-plugin-leebrary/frontend/src/components/LibraryDetailContent/components/DetailContent/DetailContent.js

```diff
@@ -0,0 +1,79 @@
+import { Box, Text, Stack, pxToRem, Badge } from '@bubbles-ui/components';
+import React from 'react';
+import { SubjectItemDisplay } from '@academic-portfolio/components';
+import { MetadataDisplay } from '../MetadataDisplay/MetadataDisplay';
+import { DETAIL_CONTENT_PROP_TYPES, DETAIL_CONTENT_DEFAULT_PROPS } from './DetailContent.constants';
+
+const DetailContent = ({
+  name,
+  description,
+  subjectsIds,
+  program,
+  metadataComponent,
+  handleCopy,
+  tags,
+  metadata,
+  icon,
+  fileType,
+  fileExtension,
+  variant,
+  variantIcon,
+  variantTitle,
+  file,
+  url,
+  classes,
+}) => (
+  <Box className={classes.tabPanel}>
+    {name && <Text className={classes.title}>{name}</Text>}
+    {description && <Text className={classes.description}>{description}</Text>}
```

Comment:
It's better to use `!!name` or `!isNil`.

If it's `0` it will print `0` instead of nothing (not this case, because it can't be 0, but it's going to print a `""` if !name or !description).

The isNil is better for numeric values where the `0` it's intended to be rendered, such as grades. With !!score it won't render the 0 value.


I think there's no need to change it.

Reply:
Done! thanks!